packages/@o3r/eslint-config/schematics/ng-add/linter/templates/project/eslint.config.js.template

```diff
@@ -0,0 +1,17 @@
+const { globalSharedConfig } = require('<%= rootRelativePath %>/eslint.config.js');
```

Comment:
The `.js` extension should be avoid as much as possible.
In a `commonjs` package it will work fine but in a ESM package it will raise an error.
you should prefer `.cjs` or `.mjs` to explicit the correct format (and I think it's time to migrate to `.mjs`)